### ISSUE TITLE
Edit/Distribute aren't recorded in the undo history

### ISSUE THREAD

**Issue by MileyHollenberg** [PixelWizards/com.pixelwizards.utilities#5]
I just noticed that once you use the Edit/Distribute X/Y/Z that you can't undo that change afterwards. It would be great to have this support for when you just want to revert to the previous positions

**Comment by gekidoslair** [comment]
yeah my understanding of how to properly handle the undo stack is pretty slim - will if I can get this in!
